evoting/frontend/package.json

```diff
@@ -1,111 +1,61 @@
 {
   "name": "evoting",
-  "version": "1.10.0",
-  "description": "A Vue.js project",
-  "author": "DEDIS",
+  "version": "0.1.1",
```

Comment:
probably 1.11.0 ? 
it seems odd to decrement the version number :)

Reply:
Ah, that's the default one. I'll bump that one again